components/network/bind/Makefile

```diff
@@ -83,6 +83,7 @@ CONFIGURE_OPTIONS += --with-tuning=default
 CONFIGURE_OPTIONS += --disable-linux-caps
 CONFIGURE_OPTIONS += READLINE_CFLAGS=-I/usr/include/readline
 CONFIGURE_OPTIONS += READLINE_LIBS="-lreadline -ltermcap"
+CONFIGURE_OPTIONS += LIBS="-lsocket"
 
 # Install.
 #
```

Comment:
When you change a package you either need to change COMPONENT_VERSION or increment/set COMPONENT_REVISION. So please add
COMPONENT_REVISION= 1

Reply:
Done.

Reply:
Where? I cannot see it in the Makefile...

Reply:
Sorry, not done.

Please also make sure you ran following make targets at least once, all of them passed and there is nothing changed in the bind component directory: `sample-manifest`, `publish`, `test`, `REQUIRED_PACKAGES`.  Thank you.

Reply:
Arrgh, miss a `git add` and I look like an idiot.

I'm chasing down a test issue (tests work fine when I cd into the build tree, but when run from the component directory, they all fail with errors resembling:
```
./aes_test: line 43: libtool_execute_magic: unbound variable
FAIL aes_test (exit status: 1)
```
 it looks like it's the result of SHELLOPTS being exported rather than my change; I see many component makefiles have a `unexport SHELLOPTS` and will be adding one to bind's.